### DISCUSSION TITLE
Question about the portal gun pedestals

### DISCUSSION THREAD

**Discussion by dsagfa** [BEEmod/BEE2-items#4108] [Tech Support / Q&A] [answered]
If I don't have a portal gun pedestal placed in my test chamber, will the test subject come out with a portal gun (like i didnt choose the portal gun pedestal) or not (like i placed the pedestal)

**Comment by TeamSpen210** [comment, marked as answer]
The way it works is that BEE checks which portals are being automatically placed by items (either portalguns or autoportals), then gives the player any remaining colours. So if you place aan orange single portalgun pedestal or an orange autoportal, the player gets a blue only gun. Note that guns combine when you pick them up, so if the player’s holding a blue only gun and picks up an orange only gun they get a dual portal device.
